gcampus/core/templates/gcampuscore/components/sidebar.html

```diff
@@ -19,7 +19,7 @@
 
             {% include "gcampuscore/components/sidebar_item.html" with icon="gcampuscore/icons/map.html" title=map_title view_name="gcampuscore:mapview" %}
             {% include "gcampuscore/components/sidebar_item.html" with icon="gcampuscore/icons/list-ul.html" title=measurements_title view_name="gcampuscore:measurements" %}
-            {% include "gcampuscore/components/sidebar_item.html" with icon="gcampuscore/icons/calculator.html" title=converter_title view_name="gcampusanalysis:od_converter" %}
+            {% include "gcampuscore/components/sidebar_item.html" with icon="gcampuscore/icons/calculator.html" title=converter_title view_name="gcampusanalysis:od_converter_overview" %}
```

Comment:
You can pass a list of other view names to `sidebar_item.html` using the `other_view_names` variable. Can you add your new view `gcampusanalysis:od_converter` to this so that the sidebar item will be highlighted even if a special OD converter view is open? You can see an example in line 25 😉

Reply:
fixed in 0279b77000609582a815774579bbdcaca4973547